src/eval/execute.rs

```diff
@@ -1,6 +1,6 @@
 use super::parse::Flags;
 use crate::eval::parse::{get_help_message, Channel, Mode};
-use crate::utils;
+use crate::utils::{self, normalize_unicode_chars};
```

Comment:
`normalize_unicode_chars` is used just once, maybe just do `utils::normalize_unicode_chars` rather than import its name?